### PR TITLE
N49: nanogltf

### PR DESCRIPTION
**Pull request by not-fl3**

### CODE REVIEW

File: content/news/049/helmet.gif
Comment (AngelOnFira) [inline content/news/049/helmet.gif:1]: We'll want to get this down to 2.5MB likely

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
Looks good after the gif size change :)

**Review by ozkriff — APPROVED** [review]
I've optimized the GIF and extended the description a bit based on social media posts, hope it's ok.

**Comment by ozkriff** [comment]
Thanks for the PR, @not-fl3. Feel free to ping me if something is wrong with my additions!

**Pull request merged**